lib/exception_hunter/error_creator.rb

```diff
@@ -12,9 +14,26 @@ class << self
       #
       # @param [HTTP_TAG, WORKER_TAG, MANUAL_TAG] tag to append to the error if any
       # @return [ExceptionHunter::Error, false] the error or false if it was not possible to create it
-      def call(tag: nil, **error_attrs)
+      def call(async_logging: Config.async_logging, tag: nil, **error_attrs)
         return unless should_create?
 
+        log_error(tag, error_attrs, async_logging)
+      rescue ActiveRecord::RecordInvalid
+        false
+      end
+
+      private
+
+      def log_error(tag, error_attrs, async_logging)
+        if async_logging
+          error_attrs = error_attrs.merge(occurred_at: Time.now)
+          ::ExceptionHunter::AsyncLoggingJob.perform_later(tag, error_attrs)
```

Comment:
How about
```suggestion
          ::ExceptionHunter::AsyncLoggingJob.perform_later(tag, error_attrs.merge(occurred_at: Time.now))
```

Too crowded? Up to you